<launch>
  <node pkg="uvc_camera" name="camera1" type="uvc_camera_node">
    <param name="device" type="string" value="/dev/video0"/>
    <param name="width" type="int" value="640"/>
    <param name="height" type="int" value="480"/>
    <param name="pixel_format" value="mjpeg"/>
    <param name="camera_frame_id" value="usb_cam"/>
    <param name="io_method" value="mmap"/>
    <param name="frame_id" value="camera"/>
    <!-- <param name="camera_info_url" type="string" value="file:///home/{user_name}/.../camera.yaml"/> -->
  </node>

  <node pkg="web_video_server" name="server1" type="web_video_server"></node>

  <node pkg="tamiya_cam_robot" name="robo1" type="wheels_listener" output="screen">
    <param name="AIN1" type="int" value="22"/>
    <param name="AIN2" type="int" value="27"/>
    <param name="PWMA" type="int" value="17"/>
    <param name="BIN1" type="int" value="13"/>
    <param name="BIN2" type="int" value="19"/>
    <param name="PWMB" type="int" value="26"/>
  </node>

  <node pkg="tamiya_cam_robot" name="light1" type="vel_color.py" output="screen"></node>

  <include file="$(find roswww)/launch/roswww.launch" /> 
  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />
</launch>
